operators/multiclusterobservability/pkg/util/backuputil.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package util
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/stolostron/multicluster-observability-operator/operators/multiclusterobservability/pkg/config"
+)
+
+func AddBackupLabelToConfigMap(c client.Client, name, namespace string) error {
+	m := &corev1.ConfigMap{}
+	err := c.Get(context.TODO(), types.NamespacedName{
+		Name:      name,
+		Namespace: namespace,
+	}, m)
+	if err != nil {
+		return err
```

Comment:
should we return the err here or just log the err?  Seems that will lead to a deadlock in some cases.

Reply:
resolved in latest commit